web/auth/tokens/jwt_test.go

```diff
@@ -0,0 +1,174 @@
+package tokens_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/golang-jwt/jwt"
+	"github.com/quickfeed/quickfeed/internal/qtest"
+	"github.com/quickfeed/quickfeed/qf"
+	"github.com/quickfeed/quickfeed/web/auth/tokens"
+)
+
+func TestNewManager(t *testing.T) {
```

Comment:
General comment about tests. I didn't look at these tests carefully in terms of the following. But I think it is possible that there are some benefits to be had from rephrasing the tests as table-driven tests. Things like:

```go
tests := []struct{
  inputA string
  inputB string
  want string
}{ ...
```

Reply:
Does it really apply to the tests in this file? I agree in general but can't see how it could be beneficial in this case.